### ISSUE TITLE
 Wrong parameter value in php section of page template if it's value equals '0'

### ISSUE THREAD

**Issue by gaabora** [octobercms/october#4185]
- OctoberCMS Build: any
- PHP Version: any

### Description:
I'm getting wrong parameter value in php section of page template if it's value equals '0'.
$this->param('paramname') returns null while 'paramname' is actually '0'

### Steps To Reproduce:
navigate to /test/0 using page template below
```
title = "test"
url = "/test/:slug"
layout = "default"
==
<?php
function onStart() {
	dump($this->param('slug')); // returns null
}
?>
==
HI! {{ this.param.slug }} // but here is ok, prints HI! 0
```

**Comment by gaabora** [comment]
it definitely cause by isset function in getParameter() of Cms\Classes\Router
```
public` function getParameter($name, $default = null)
    {
        if (isset($this->parameters[$name]) && !empty($this->parameters[$name])) {
            return $this->parameters[$name];
        }

        return $default;
    }
```
so how about
`if (array_key_exists($name, $this->parameters)) { ...`?

**Comment by bennothommo** [comment]
Fixed by #4657.
